lib/theme/sizing.dart

```diff
@@ -1,15 +1,22 @@
 class Sizing {
+  /// represent size of 8.0
```

Comment:
Why not Spacing and define this Sizing

Reply:
We don't need to define Sizing class just serves the same purpose as Spacing. Remove all the variables that we haven't used yet to clarify.

Reply:
@RonTran636 @sieuhuflit Sizing definitely has another meaning with Spacing, even if some of their value might be the same with Spacing, but the meaning is different. FYI